src/styles/price.scss

```diff
@@ -0,0 +1,8 @@
+.price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: $defaultWidth;
```

Comment:
Ensure that the variable '$defaultWidth' is defined in your SCSS files. If it is not defined, the SCSS compilation will fail.